src/megameklab/com/util/MenuBarCreator.java

```diff
@@ -817,60 +722,58 @@ private void jMenuInsertImageFile_actionPerformed() {
 
             if (fDialog.getFile() != null) {
                 String relativeFilePath = new File(fDialog.getDirectory() + fDialog.getFile()).getAbsolutePath();
-                relativeFilePath = "." + File.separatorChar + relativeFilePath.substring(new File(System.getProperty("user.dir").toString()).getAbsolutePath().length() + 1);
+                relativeFilePath = "." + File.separatorChar + relativeFilePath.substring(new File(System.getProperty("user.dir")).getAbsolutePath().length() + 1);
```

Comment:
Likewise, is there a relative path tool? (granted this code has probably worked just fine for years)

Reply:
All I was doing here was removing the extraneous `toString()` call on a String expression. I didn't worry about the rest of the line, which has proved to be functional if not necessarily ideal.